### PR TITLE
Use of baseline widths in Twitch Amp, Twitch AUC, Twitch Velocities

### PR DESCRIPTION
**Pull request by luciipak**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Changelog for Pulse3D
 =====================
 
+0.32.7 (2023-05-01)
```
Comment (tannermpeterson) [inline CHANGELOG.rst:4]: We should bump this to 0.33.0 since we're changing a bunch of metrics

Comment (luciipak) [reply CHANGELOG.rst:4]: Cool I was wondering about that

File: src/pulse3D/metrics.py [hunk 2]
```diff
@@ -565,18 +574,19 @@ def calculate_area_under_curve(
...
-        width_percent = 90  # what percent of repolarization to use as the bottom limit for calculating AUC
         estimates_dict: Dict[int, Union[int, float]] = dict()
 
         rising_x_values = coordinate_df["time"]["contraction"].T.to_dict()
         falling_x_values = coordinate_df["time"]["relaxation"].T.to_dict()
 
         for iter_twitch_peak_idx in twitch_indices.keys():
-            start_timepoint = rising_x_values[iter_twitch_peak_idx][width_percent]
-            stop_timepoint = falling_x_values[iter_twitch_peak_idx][width_percent]
+            start_timepoint = rising_x_values[iter_twitch_peak_idx][baseline_widths[0]]
```
Comment (tannermpeterson) [inline src/pulse3D/metrics.py:588]: Should one of these be subtracted from 100?

Comment (luciipak) [reply src/pulse3D/metrics.py:588]: yes, it is above on line 531

**Review by tannermpeterson — APPROVED** [review]

**Pull request merged**